docs/healpix.rst

```diff
@@ -0,0 +1,55 @@
+**************
+HEALPIX images
```

Comment:
Could you use an example data file in `healpix.rst` that the user can get?

The descriptions are good, but it would be even much nicer if I could actually copy & paste the code into an IPython notebook and start playing with this ...
See http://www2.mpia-hd.mpg.de/~robitaille/astropy/reproject_docs/healpix.html